pkg/env/terraform_env.go

```diff
@@ -148,6 +168,53 @@ func (e *TerraformEnvPrinter) Print() error {
 // Private Methods
 // =============================================================================
 
+// getTerraformOutputs retrieves Terraform outputs from the specified project path.
+// It executes 'terraform output -json' and converts the outputs into environment variables.
+// Each output is prefixed with 'TF_VAR_' to make it available as a Terraform variable.
+// Returns a map of environment variable names to their formatted values, or an error if the command fails.
+func (e *TerraformEnvPrinter) getTerraformOutputs(projectPath string) (map[string]string, error) {
+	cmd := e.shims.Command("terraform", "-chdir="+projectPath, "output", "-json")
```

Comment:
For command/exec specifically, the `shell` package is used when executing commands. This approach abstracts execution so that in the future, we could execute commands against other types of shells--docker exec, SSH, etc.

You can find examples of this elsewhere, but it this will end up being something like `shell.ExecSilent("terraform", "-chdir=...")